promscale/recommendations/resource-recomm.md

```diff
@@ -45,23 +45,31 @@ remote_write:
 
 Compute recommendations for the Promscale connector and TimescaleDB are:
 
-|Ingestion Rate|Connector CPU|Connector Memory|DB CPU|DB Memory|DB connections|
-|-|-|-|-|-|-|
-|10k samples/sec|0.5|250 MB|1|4 GB|1|
-|50k samples/sec|2|700 MB|4|16 GB|2|
-|100k samples/sec|4|2 GB|8|32 GB|4|
-|200k samples/sec|8|4.5 GB|16|64 GB|8|
+|Ingestion Rate|Active Series|Connector CPU|Connector Memory|DB CPU|DB Memory|DB connections|DB Shared Buffers|
+|-|-|-|-|-|-|-|-|
+|10k samples/sec|500K|0.5|2 GB|1|12 GB|1|8 GB|
+|50k samples/sec|2M|2|8 GB|4|32 GB|2|16 GB|
+|100k samples/sec|3M|2|8 GB|8|64 GB|4|24 GB|
+|200k samples/sec|5M|4|16 GB|16|64 GB|8|40 GB|
+
+<highlight type="note">
+You can use `shared_buffers` PostgreSQL configuration to set `DB Shared Buffers`.
```

Comment:
@arajkumar how about `You should use `shared_buffers` in PostgreSQL configuration to set `DB Shared Buffers`.`